src/test/resources/reference.conf

```diff
@@ -55,10 +55,10 @@ proxy {
 
 clusterFiles {
   configFolderPath = "src/test/resources/"
-  jupyterServerCrt = "test.crt"
-  jupyterServerKey = "test.key"
-  jupyterRootCaPem = "test.pem"
-  jupyterRootCaKey = "test.key"
+  jupyterServerCrt = "test-server.crt"
```

Comment:
These files have to exist now because `MockGoogleStorageDAO` checks for existence :)